src/librustdoc/html/static/rustdoc.css

```diff
@@ -902,7 +907,6 @@ body.blur > :not(#help) {
 	font-size: initial;
 	position: absolute;
 	right: 0;
-	top: 0;
```

Comment:
Instead of removing this, please add `position: relative;` to `.small-section-header {`.

Reply:
Sure, thanks for the hint

Reply:
I mistakenly force-pushed, it should have been an extra commit as I read from @rust-highfive  ...my bad!

Reply:
Not an issue. :)